core/src/Revolution/Processors/Security/Login.php

```diff
@@ -170,29 +170,31 @@ public function checkIsBlocked()
         /** @var modUserProfile $profile */
         $profile = $this->user->Profile;
 
-        if ($profile->get('failed_logins') >= $this->modx->getOption('failed_login_attempts') && $profile->get('blockeduntil') > time()) {
-            return $this->modx->lexicon('login_blocked_too_many_attempts');
-        }
-
         if ($profile->get('failedlogincount') >= $this->modx->getOption('failed_login_attempts')) {
```

Comment:
By removing the check above and instead letting it fall back to this `if {}` block, the blocked time will be extended each time the user tries to login while they're still blocked - is that intentional?

Reply:
Not necessarily a requirement for this PR to be approved but It would be really great to have a Unit Test for this.

Reply:
> By removing the check above and instead letting it fall back to this `if {}` block, the blocked time will be extended each time the user tries to login while they're still blocked - is that intentional?

This will never happen, because if the user is blocked, the count of login attempts will stop. By the way, this condition will always be false. And not just because the "failed_logins" field doesn't exist.